crates/modelardb_bulkloader/src/main.rs

```diff
@@ -52,14 +52,22 @@ async fn main() -> Result<()> {
         print_usage_and_exit_with_error();
     }
 
-    // unwrap() is safe as the path of the binary is dropped.
-    args.next().unwrap();
-
-    // unwrap() on args.next() is safe as args contains at least five elements.
-    let operation = args.next().unwrap();
-    let input_path = args.next().unwrap();
-    let output_path = args.next().unwrap();
-    let table_name = args.next().unwrap();
+    // Drop the path of the binary.
+    args.next()
+        .expect("args should contain at least five arguments.");
```

Comment:
Instead of duplicating the string, maybe assign it to a `str` variable and pass it to each.